test/validation_test.go

```diff
@@ -459,6 +470,215 @@ func testAlertRules(t *testing.T) {
 	}
 }
 
+type resourceMeta struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+}
+
+// shrinked version of github.com/VictoriaMetrics/operator/api/v1beta1.VMAgent
+type VMAgent struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              struct {
+		ServiceScrapeSelector          *metav1.LabelSelector `json:"serviceScrapeSelector,omitempty"`
+		ServiceScrapeNamespaceSelector *metav1.LabelSelector `json:"serviceScrapeNamespaceSelector,omitempty"`
+		PodScrapeSelector              *metav1.LabelSelector `json:"podScrapeSelector,omitempty"`
+		PodScrapeNamespaceSelector     *metav1.LabelSelector `json:"podScrapeNamespaceSelector,omitempty"`
+		ProbeSelector                  *metav1.LabelSelector `json:"probeSelector,omitempty"`
+		ProbeNamespaceSelector         *metav1.LabelSelector `json:"probeNamespaceSelector,omitempty"`
+	} `json:"spec"`
+}
+
+// shrinked version of github.com/VictoriaMetrics/operator/api/v1beta1.VMAlert
+type VMAlert struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              struct {
+		RuleSelector          *metav1.LabelSelector `json:"ruleSelector,omitempty"`
+		RuleNamespaceSelector *metav1.LabelSelector `json:"ruleNamespaceSelector,omitempty"`
+	} `json:"spec"`
+}
+
+func testVMCustomResources(t *testing.T) {
+	const vmBaseDir = manifestDir + "monitoring/base/victoriametrics"
```

Comment:
```suggestion
	vmBaseDir := filepath.Join(manifestDir, "monitoring", "base", "victoriametrics")
```

`filepath.Join(manifestDir, "monitoring/base/victoriametrics")` is acceptable for me though it would be less portable.
Aside from portability, `filepath.Join()` allows us not to be worried about leading/trailing slashes as used in `dirPath + "/file"`.